### PR TITLE
MET-2221 Added grid padding elements / normalised media queries

### PR DESCRIPTION
**Pull request by andyjmaclean**

### CODE REVIEW

File: src/app/dashboard/executionsgrid/gridrow/gridrow.component.scss [hunk 1]
```diff
@@ -92,6 +92,71 @@ $row-border-height-min: 3px;
...
+
+  &.dates {
+    grid-column-start: sub-date-start;
+    grid-column-end: sub-date-end;
+  }
+
+  /* set animations */
+  .orb-status {
+    margin-left: 0.75em;
+    //margin-right: 0;
```
Comment (mirjamverloop) [inline src/app/dashboard/executionsgrid/gridrow/gridrow.component.scss]: If unused, remove this.

File: src/app/dataset/executions-data-grid/executions-data-grid.component.scss [hunk 2]
```diff
@@ -30,36 +43,41 @@ $width-orb: 3em;
...
-    margin-left: auto;
-    width: 8em;
+    margin-left: 0;
+    width: auto;
   }
 
   .table-grid-row-start {
-    display: inline-block;
-    width: $width-orb;
+    //display: inline-block;
```
Comment (mirjamverloop) [inline src/app/dataset/executions-data-grid/executions-data-grid.component.scss]: If unused, clean it up.

File: src/app/dataset/executions-data-grid/executions-data-grid.component.scss [hunk 3]
```diff
@@ -3,6 +3,12 @@
 
 $width-orb: 3em;
 
+.buttonplaceholder {
+  &:empty {
+    display: none;
+  }
```
Comment (mirjamverloop) [inline src/app/dataset/executions-data-grid/executions-data-grid.component.scss]: If have seen this a couple of times in your code, could you say everything that is :empty should not be displayed? In other words, could this be a generic rule?

Comment (andyjmaclean) [reply src/app/dataset/executions-data-grid/executions-data-grid.component.scss]: In some contexts.  When the elements are stacked empty elements create irregular spacing.  When the elements are in the grid the empty cells have to be displayed for spacing.

File: src/app/dataset/executions-data-grid/executions-data-grid.component.scss (hunk 2 above)
Comment (andyjmaclean) [reply src/app/dataset/executions-data-grid/executions-data-grid.component.scss]: Removed

File: src/app/dashboard/executionsgrid/gridrow/gridrow.component.scss (hunk 1 above)
Comment (andyjmaclean) [reply src/app/dashboard/executionsgrid/gridrow/gridrow.component.scss]: Removed

**Pull request merged**